docs/core/run-time-config/garbage-collector.md

```diff
@@ -137,8 +209,20 @@ For more information about some of these settings, see the [Middle ground betwee
 | **runtimeconfig.json** | `System.GC.HeapHardLimit` | *decimal value* | .NET Core 3.0 |
 | **Environment variable** | `COMPlus_GCHeapHardLimit` | *hexadecimal value* | .NET Core 3.0 |
 
+Example:
+
+```json
+{
+   "runtimeOptions": {
+      "configProperties": {
+         "System.GC.HeapHardLimit": 80000
```

Comment:
I know this is supposed to be an example only but it'd be good to have a more realistic value :) like 209715200 which is 200MiB.